datasets/plugins/batch/zillow_dataset_path.py

```diff
@@ -0,0 +1,18 @@
+import os
+from pathlib import Path
+
+from datasets.plugins import BatchDatasetPlugin
+
+
+def _get_batch_dataset_path(dataset: BatchDatasetPlugin) -> str:
```

Comment:
Guessing you already know this but what's the plan to remove this Zillow specific code from the Open Source implementation?

Reply:
Adding comment: `AIP-5405 Create zillow-datasets library for Zillow Offline dataset policies`